src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -25,9 +25,9 @@ public FeedbackResponseRow(int fbIndex, int personIndex, String personType,
         FeedbackQuestionDetails questionDetails = question.getQuestionDetails();
         this.questionNumber = question.questionNumber;
         this.questionText = results.getQuestionText(questionId);
-        this.questionMoreInfo = questionDetails.getQuestionAdditionalInfoHtml(this.questionNumber,
-                                                                              personType + "-" + personIndex
-                                                                                         + "-session-" + fbIndex);
+        this.questionMoreInfo =
+                questionDetails.getQuestionAdditionalInfoHtml(this.questionNumber,
+                                                              personType + "-" + personIndex + "-session-" + fbIndex);
```

Comment:
Same as somewhere above:
Maybe put the method call on the first line, and put the arguments on the second line.